frontend/src/app/core/helpers/time-zone-helper.ts

```diff
@@ -1,18 +1,82 @@
-import { formatDate } from '@angular/common';
 import { ITimeZone } from '@core/models/ITimeZone';
-import { ILocalUser } from '@core/models/IUser';
+
+export const millisInHour = 3600 * 1000;
+export const millisInMinute = 60 * 1000;
+
+export const setTimeZone = (timeZone: ITimeZone) => {
+    localStorage.setItem('timeZone', JSON.stringify({ nameValue: timeZone.nameValue, timeValue: timeZone.timeValue }));
```

Comment:
Can we write `JSON.stringify(timeZone)`?

Reply:
1. If we try to get time zone manually we can gather such info
![image](https://user-images.githubusercontent.com/34366638/191805557-1a7af6dd-1543-4a94-9747-061c6b35e1c6.png)
using 
```
    const timeZoneValue = `${hours}:${minutes}`;
    const timeZoneName = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

2. `<ng-time-zone-picker>` on the other hand, also, returns time zone object, but it has more properties
![image](https://user-images.githubusercontent.com/34366638/191806646-cd1397dd-3047-4870-8973-010f8ed5a682.png)

Thus, when `getDefaultTimeZone()` method is called from the page without `<ng-time-zone-picker>` only limited data (from 1 screenshot) will be saved to _localStorage_. 

I think it's bad, because someone may stumble upon extended variant in _localStorage_ and use it, while no one can guarantee, the values will be there. As well as we simply don't need extended version values